formats/completed_transaction/publisher/details.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "presentation_toggles"
```

Comment:
Why is this and `promotion_choice` required?

There are live examples where no promotions display.